e2e/davinci-app/main.ts

```diff
@@ -11,13 +11,11 @@ import flowLinkComponent from './components/flow-link.js';
 import socialLoginButtonComponent from './components/social-login-button.js';
 
 const config = {
-  // clientId: '7544ef78-5c01-48c4-a9a5-6338c10c7f87',
-  clientId: '724ec718-c41c-4d51-98b0-84a583f450f9', // Luis' tenant
+  clientId: '724ec718-c41c-4d51-98b0-84a583f450f9',
```

Comment:
Do we want these to be PR'd or should we begin a migration to secrets / .env file?

Honestly - if we don't need these to be secrets I think that's the best way to go about it.

Without a more scaleable solution for us as we scale tenants and packages and what not, its going to be really hard to maintain various configuration settings (unless we have some secret solution that we currently do not have).

Reply:
This isn't a secret, just a client ID in our test tenant. Though, I want to switch these tests to use the mock API we have, so this will probably go away.